btcstaking-tracker/btcslasher/bootstrapping.go

```diff
@@ -53,24 +70,25 @@ func (bs *BTCSlasher) Bootstrap(startHeight uint64) error {
 				continue
 			}
 
-			// slash this finality provider's all BTC delegations whose slashing tx's input is still spendable
-			// on Bitcoin
+			// Attempt to slash the finality provider
 			if err := bs.SlashFinalityProvider(fpBTCSK); err != nil {
 				bs.logger.Errorf("failed to slash finality provider %s: %v", fpBTCPKHex, err)
 				accumulatedErrs = multierror.Append(accumulatedErrs, err)
 
 				continue
 			}
+
+			lastSlashedHeight = evidence.BlockHeight
```

Comment:
```suggestion
            if lastSlashedHeight < evidence.BlockHeight {
			    lastSlashedHeight = evidence.BlockHeight
            }
```
Not sure if this check is needed as I don't know whether the returned evidences are ordered